webhook/server/server.go

```diff
@@ -121,9 +109,9 @@ func (s *WebhookServer) ListenAndServe() error {
 	}
 }
 
-func (s *WebhookServer) runAdmissionWebhookListenAndServe(client *client.Client, handler http.Handler, validationResources []admission.Resource, mutationResources []admission.Resource) error {
-	apply := client.Apply.WithDynamicLookup()
-	client.Core.Secret().OnChange(s.context, "secrets", func(key string, secret *corev1.Secret) (*corev1.Secret, error) {
+func (s *WebhookServer) runAdmissionWebhookListenAndServe(clients *client.Clients, handler http.Handler, validationResources []admission.Resource, mutationResources []admission.Resource) error {
```

Comment:
NIT: No need to input `clients *client.Clients`. Same below

Reply:
Updated. Thank you.